docs/src/content/docs/ru/guides/i18n.mdx

```diff
@@ -393,12 +393,13 @@ const arabicDirection = Astro.locals.t.dir('ar');
 
 Вы можете использовать [`Astro.currentLocale`](https://docs.astro.build/ru/reference/api-reference/#astrocurrentlocale) для получения текущей локали в компонентах `.astro`.
 
-Следующий пример считывает текущую локаль и использует её для создания ссылки на страницу о сайте на текущем языке:
+Следующий пример считывает текущую локаль и использует её с помощью хелпера [`getRelativeLocaleUrl()`](https://docs.astro.build/ru/reference/modules/astro-i18n/#getrelativelocaleurl) для генерации ссылки на страницу «О сайте» на текущем языке:
 
 ```astro
 ---
 // src/components/AboutLink.astro
+// import { getRelativeLocaleUrl } from 'astro:i18n';
```

Comment:
```suggestion
import { getRelativeLocaleUrl } from 'astro:i18n';
```

Looks like an extra comment, I'll remove it so we can merge the new translations.